vocabularies/scripts/update-csv.py

```diff
@@ -0,0 +1,90 @@
+# This script updates the keys in a given file with the provided values.
+#
+# Depends on Python 3
+#
+# Install dependencies:
+#   pip3 install -r requirements.txt
+#
+#
+#   Invocation:
+#   cd vocabularies/scripts
+#   python3 update-csv.py locations spaghetti.csv
+#
+#   Params:
+#    - [which]: Name of csv file to update
+#               (e.g. 'recapCustomerCodes', 'organizations').
+#    - [updateInfo] Name of csv file with values to update
+import sys
+import csv
+
+
+def csv_to_dict(file_name: str) -> dict:
+    output_dict = {}
+    row_data = {}
+
+    try:
+        with open(file_name, 'r') as f:
+            raw_csv = f.readlines()
+
+        header = raw_csv[:1][0].strip().split(',')
+        for _, row_as_string in enumerate(raw_csv[1:]):
+            exploded_row = row_as_string.replace("\n", "").split(",")
+            primary_key = exploded_row[0]
+            for index, nypl_core_key in enumerate(header):
+                try:
+                    row_data[nypl_core_key] = exploded_row[index]
+                except IndexError:
+                    # Not every row has the right number of commas. This should
+                    # be addressed by running the script, as empty fields will
+                    # be assigned as a side effect.
+                    pass
+            output_dict[primary_key] = row_data
+            row_data = {}
+
+        return output_dict
+
+    except Exception as e:
+        print(e)
+        return {"Error": e}
```

Comment:
Hmm if we fail to convert the csv to a dict, recommend just raising the error unless there are things we can process with this error dict.